openpype/hosts/maya/plugins/load/load_audio.py

```diff
@@ -76,7 +84,7 @@ def update(self, container, representation):
             project_name, version["parent"], fields=["parent"]
         )
         asset = get_asset_by_id(
-            project_name, subset["parent"], fields=["parent"]
+            project_name, subset["parent"], fields=["parent", "data"]
```

Comment:
This would also be a possibility, right? Or doesn't AYON support that?
```suggestion
            project_name, subset["parent"], fields=["parent", "data.frameStart", "data.frameEnd"]
```